src/containers/profile/Profile.css

```diff
@@ -0,0 +1,29 @@
+.profileContainer {
+    border-radius: 1rem;
+    position: absolute;
+    top: 80%;
+    width: 70%;
+    background-color: white;
+  }
+  
+  .extraComponents {
+    color: white;
+  }
+  
+  .extraComponents span,
+  .extraComponents p {
+    color: #c1c3c7;
+  }
+  
+  .jobTitle {
+    font-size: 0.6em;
+    margin-left: 8px;
+    font-weight: 400;
+  }
+  
+  .brief {
```

Comment:
the ```  .brief ``` class name might be confusing, what if we other brief description in other components so we'll get confused between this and that. I think you can name something like ```.profileBrief``` or another descriptive name